requirements.txt

```diff
@@ -6,6 +6,7 @@ pyvisfile
 pymetis
 importlib-resources
 psutil
+gmsh
```

Comment:
As we discussed, this has the potential to break installs where gmsh cannot be installed by `pip`.  One platform where it *definitely* doesn't work at all is *Lassen* - because there are no compatible `gmsh` packages for that platform, and it fails to install using `pip`.   

One platform that is iffy is *Quartz* because as we've noted there it is required to be installed by `conda` and not `pip` in order to get a working install.  

The only platform that seems to be OK with `gmsh` installed via `pip` is the CI VM.

Any ideas on what to do about this?   CC: @inducer, @matthiasdiener

Reply:
I can remove this if needed for now, but without it the doublemach-mpi.py case will fail CI. I added it for now to make sure it would pass CI if it had gmsh. Another option would be to include the mesh as a data file so we don't need gmsh to generate it, but I wasn't sure if we want a mesh file in the repository. Maybe the mesh file could even exist in another data repository that the CI pulls if we don't want it to live in the mirgecom repo.

Reply:
I think you should leave it until we can discuss how we should handle it. It is likely there will be something that can be done to get CI to test it as well as fix installs.

Reply:
This is resolved.  pip installs gmsh on all the platforms I tried so far including Quartz and Lassen.  Time travel duplicate paradox resolved.